Documentation/gettingstarted/policy-creation.rst

```diff
@@ -85,6 +96,28 @@ verdicts using ``cilium monitor -t policy-verdict``. We'll be monitoring for
 inbound traffic towards the deathstar to identify that traffic and determine
 whether to extend the network policy to allow that traffic.
 
+Apply a default-deny policy:
+
+.. literalinclude:: ../../examples/minikube/sw_deny_policy.yaml
+
+CiliumNetworkPolicies match on pod labels using an "endpointSelector" to identify
+the sources and destinations to which the policy applies. The above policy denies
+traffic sent to any pods with label (``org=empire``). Due to the Policy Audit Mode
+enabled above (either for the entire daemon, or for just the ``deathstar`` endpoint),
+the traffic will not actually be denied but will instead trigger policy verdict
+notifications.
+
+To apply this policy, run:
+
+.. code-block:: shell-session
```

Comment:
This will also need to be a parsed-literal in order for the `\ |SCM_WEB|` link to show up correctly.

Reply:
reverted "code-block..."